pkg/pbflow/proto.go

```diff
@@ -100,13 +102,34 @@ func FlowToPB(fr *model.Record, s *ovnobserv.SampleDecoder) *Record {
 	}
 	if s != nil {
 		seen := make(map[string]bool)
+		pbflowRecord.NetworkEventsMetadata = make([]*NetworkEvent, 0)
 		for _, metadata := range fr.Metrics.NetworkEvents {
+			var pbEvent NetworkEvent
 			if !model.AllZerosMetaData(metadata) {
 				if md, err := s.DecodeCookie8Bytes(metadata); err == nil {
 					protoLog.Debugf("Network Events Metadata %v decoded Cookie: %v", metadata, md)
-					if !seen[md] {
-						pbflowRecord.NetworkEventsMetadata = append(pbflowRecord.NetworkEventsMetadata, md)
-						seen[md] = true
+					acl, ok := md.(*ovnmodel.ACLEvent)
```

Comment:
md.String() is called three times below I guess it could be set in a var instead